pygmt/tests/test_clib_loading.py

```diff
@@ -18,13 +18,6 @@ def test_load_libgmt():
     check_libgmt(load_libgmt())
 
 
-def test_load_libgmt_fail():
-    "Test that loading fails when given a bad library path."
-    env = {"GMT_LIBRARY_PATH": "not/a/real/path"}
-    with pytest.raises(GMTCLibNotFoundError):
-        load_libgmt(env=env)
-
-
```

Comment:
Maybe something like this (will need to `import os` on the top, and reset GMT_LIBRARY_PATH too)
```python
def test_load_libgmt_fail():
    "Test that loading fails when given a bad library path."
    os.environ["GMT_LIBRARY_PATH"] = "not/a/real/path"
    with pytest.raises(GMTCLibNotFoundError):
        load_libgmt()
```

Reply:
Have to save the original value and revert it back after the test.

Fixed in 9251d7c1196a35f6ed9a73718e710fc91952d075.